website/docs/troubleshooting/troubleshooting-podman.md

```diff
@@ -233,3 +233,26 @@ This might appear when either:
 3. Restart the Podman machine to recreate and activate the default Docker socket path.
 
 _Note:_ If Docker Desktop is started again, it will automatically re-alias the default Docker socket location and the Podman compatibility warning will re-appear.
+
+## Uninstalling Podman Desktop preserves the old configuration data  
+
+#### Issue
+
+After uninstalling Podman Desktop, the configuration data persists even though it is not needed for a fresh installation.
+
+
+#### Solution
+
+**_Deleting Podman Desktop configuration_**
+
+- Perform one of the following steps based on your operating system:
+   - On macOS, go to the _~/.local/share/containers/podman-desktop/_ directory and delete the _configuration_ folder.
+   - On Windows and Linux, go to the _~/.local/share/containers/podman-desktop/_ directory and delete the _configuration_ folder.
```

Comment:
repeat "on" on each sentence, maybe just `macOS:` or something like that? 

small nitpick though, don't mind either way, but unsure what the correct way would be grammatically.

Reply:
same here, use \` for showing dir